src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -255,19 +255,28 @@ The prepare-processing and InSAR processing are combined in the insar_tps_burst
  - topsapp.TopsappBurstConfig # Set the configure file
 
 ## InSAR processing ##
-ISCE2 InSAR processing includes the list of steps:
+ISCE2 InSAR processing generally includes the list of steps:
 ['startup', 'preprocess', 'computeBaselines', 'verifyDEM', 'topo', 'subsetoverlaps', 'coarseoffsets', 'coarseresamp',
  'overlapifg', 'prepesd', 'esd', 'rangecoreg', 'fineoffsets', 'fineresamp', 'ion', 'burstifg', 'mergebursts', 'filter',
  'unwrap', 'unwrap2stage', 'geocode', 'denseoffsets', 'filteroffsets', 'geocodeoffsets']
 
-This workflow includes following processes:
-- topsapp.run_topsapp_burst # Run ISCE2 from step 'start' to step 'preprocess'
-- topsapp.swap_burst_vrts # Switch the reference and secondary bursts
-- topsapp.run_topsapp_burst # Run ISCE2 steps from 'computeBaselines' to 'unwrap2stage'
-- topsapp.run_topsapp_burst # Run ISCE2 step 'geocode'
+This workflow includes following processes (refers to https://github.com/isce-framework/isce2-docs/blob/66a66ff31f4673eed9305f06982aaf4302a163cc/Notebooks/UNAVCO_2020/TOPS/topsApp.ipynb)
+
```

Comment:
Is there any reason not to reference the specific ISCE functions used for each step?

Reply:
In ISCE2, it calls steps. Users can ask the isce to process the specific steps by giving the definition of steps they want to do. I guess we don't need dive in isce2 code the find out the function names.

Reply:
For the InSAR GAMMA products, the specific commands are listed, which has been helpful for troubleshooting (even within the team). I just assumed that each one of the steps listed in the workflow would have a particular function associated (i.e. on line 266 where the specific function name was removed from the description).

Reply:
Or do you just want users to go to the notebook that is now referenced? Is that a clear indication of what we're doing in our code?

Reply:
Addressed with commit https://github.com/ASFHyP3/hyp3-isce2/pull/88/commits/931e7ef8a47c891c9bb117f0ef4ddc50fe8551d9